openhub/portfolio_projects.py

```diff
@@ -0,0 +1,80 @@
+import json
+import requests
+import logging
+import xmltodict
+
+from community.git import get_owner
+from openhub.oh_token import OH_TOKEN
+from openhub.models import PortfolioProject, PortfolioProjectActivity
+
+
+def get_portfolio_projects_data():
+    n = 100
+    data1 = []
+    for i in range(1, n):  # Ignore PyFlakesBear
+        import_url = 'https://www.openhub.net/orgs/' + get_owner() \
+            + '/projects.xml?api_key=' + OH_TOKEN + '&page=' + str(i)
+        resp = requests.get(import_url)
+        try:
+            jsonString = json.dumps(xmltodict.parse(resp.content),
+                                    indent=4)
+            json_object = json.loads(jsonString)
+            data = json_object['response']['result'
+                                           ]['portfolio_projects']['project']
+        except Exception as ex:
+            logging.debug(ex)
+            break
+        data1 = data1 + data
+    return data1
+
+
+def import_data(project):
+    name = project.get('name', None)
+    activity = project.get('activity', None)
+    i_use_this = project.get('i_use_this', None)
+    community_rating = project.get('community_rating', None)
+    org = get_owner()
+    commits = project.get(
+        'twelve_mo_activity_and_year_on_year_change', None).get('commits', None)
+    change_in_commits = project.get(
+        'twelve_mo_activity_and_year_on_year_change', None).get(
+        'change_in_commits', None)
+    percentage_change_in_commits = project.get(
+        'twelve_mo_activity_and_year_on_year_change', None).get(
+        'percentage_change_in_commits', None)
+    contributors = project.get(
+        'twelve_mo_activity_and_year_on_year_change', None).get(
+        'contributors', None)
+    change_in_contributors = project.get(
+        'twelve_mo_activity_and_year_on_year_change', None).get(
+        'change_in_contributors', None)
+    percentage_change_in_committers = project.get(
+        'twelve_mo_activity_and_year_on_year_change', None).get(
+        'percentage_change_in_committers', None)
+
+    try:
+        (cr, create) = PortfolioProjectActivity.objects.get_or_create(
+            commits=commits,
```

Comment:
you can probably do `**project['twelve_mo_activity_and_year_on_year_change']`

that will auto fill all these fields, as the field names are the same as the dict member names